src/server.ts

```diff
@@ -32,20 +29,31 @@ app.post('/scan/configure', (request: Request, response: Response) => {
   response.json({ status: 'ok' })
 })
 
-app.post('/scan/scan', (_request: Request, response: Response) => {
+app.post('/scan/scanBatch', (_request: Request, response: Response) => {
   doScan()
+    .then(() => {
+      response.json({ status: 'ok' })
+    })
+    .catch(err => {
+      response.json({ status: `could not scan ${err}` })
+    })
+})
+
+app.post('/scan/invalidateBatch', (_request: Request, response: Response) => {
   response.json({ status: 'ok' })
 })
```

Comment:
Shouldn't this method do some invalidating?

Reply:
eventually yes :)